packages/shorebird_cli/lib/src/commands/release_command.dart

```diff
@@ -117,11 +118,23 @@ Did you forget to run "shorebird init"?''',
 
     if (releaseVersionArg == null) logger.info('');
 
-    final releaseVersion = releaseVersionArg ??
-        logger.prompt(
-          'What is the version of this release?',
-          defaultValue: versionString,
+    Version? releaseVersion;
+    var releaseVersionInput = releaseVersionArg;
+    while (releaseVersion == null) {
+      releaseVersionInput = releaseVersionInput ??
+          logger.prompt(
+            'What is the version of this release?',
+            defaultValue: versionString,
+          );
+      try {
+        releaseVersion = Version.parse(releaseVersionInput);
+      } catch (error) {
+        logger.err(
+          '"$releaseVersionInput" is not a valid version number (see https://semver.org/)',
```

Comment:
I think if I'm seeing this number, I might ask "why do I care?  Why does it need to be semver?  Why are you, my release system telling me how I can name my releases?" :)

Reply:
Yeah that's a good point as well. We might want to leave this as is and instead improve the experience of managing releases so you can delete/edit releases.